tests/tutorials/test_tutorials.py

```diff
@@ -114,10 +113,10 @@ def test_gluon_save_load_params():
 
 def test_gluon_hybrid():
     assert _test_tutorial_nb('gluon/hybrid')
-
+"""
```

Comment:
can we add a comment linking the related issue here. same for other commented code.